gatsby/static/js/how-it-works.js

```diff
@@ -483,14 +483,14 @@ function nextStage() {
 
     var dissolveTime = 500;
 
-    d3.select("#legend" + stageIndex).style("opacity", 1)
+    d3.select("#legend" + stageIndex).style("display", "block")
         .transition()
         .duration(dissolveTime)
-        .style("opacity", 1e-6);
-    d3.select("#legend" + (stageIndex + 1)).style("opacity", 1e-6)
+        .style("display", "none");
```

Comment:
The opacity transition is lost here. We can still have the best of both worlds though.

Or if you can't figure it out, we can always use `pointer-events: none;` on hidden panels so you can click through them.

Reply:
HI @MadLittleMods , thanks for the feedback. I have used the second option of `pointer-events: none`. If you could kindly review, thanks.